controllers/database_controller.go

```diff
@@ -74,15 +84,8 @@ func (r *DatabaseReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 
 	NDBInfo := database.Spec.NDB
 	username, password, caCert, err := getNDBCredentialsFromSecret(ctx, r.Client, NDBInfo.CredentialSecret, req.Namespace)
-	if err != nil || username == "" || password == "" {
-		var errStatement string
-		if err == nil {
-			errStatement = "NDB username or password cannot be empty"
-			err = fmt.Errorf("empty NDB credentials")
-		} else {
-			errStatement = "An error occured while fetching the NDB Secrets"
-		}
-		log.Error(err, errStatement)
+	if err != nil {
+		r.recorder.Event(database, "Warning", "InvalidNDBCredentials", err.Error())
```

Comment:
"InvalidNDBCredentials" -> "Error: InvalidNDBCredentials"

Reply:
This applies to all the error cases where you are emitting an event.

Reply:
I have added "Error:" to all warning events. 

Kubernetes supports only two event types: Info and Warning. 
https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/event-v1/